src/module/overrideModule.js

```diff
@@ -39,21 +39,32 @@ var OverrideModule = class OverrideModule {
         const _shouldAnimateActor = WindowManager.prototype._shouldAnimateActor;
         WindowManager.prototype._shouldAnimateActor = function () {
             let actor = arguments[0];
+            let types = arguments[1];
+
+            if (
+                !actor ||
+                this._skippedActors.delete(actor) ||
+                !this._shouldAnimate() ||
+                !actor.get_texture()
+            ) {
+                return false;
+            }
+
             if (actor.manipulateByMs && !actor.completeIsRequested) {
                 actor.completeIsRequested = true;
                 return true;
             }
-            return _shouldAnimateActor.apply(this, arguments);
+            return types.includes(actor.meta_window.get_window_type());
```

Comment:
Closer to original source: https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/gnome-3-36/js/ui/windowManager.js